servers/rendering/renderer_rd/renderer_scene_environment_rd.h

```diff
@@ -34,26 +34,50 @@
 #include "servers/rendering/renderer_scene_render.h"
 #include "servers/rendering/rendering_device.h"
 
+// TODO Make members private where possible
+
 class RendererSceneEnvironmentRD {
 private:
 	static uint64_t auto_exposure_counter;
 
-public:
 	// BG
 	RS::EnvironmentBG background = RS::ENV_BG_CLEAR_COLOR;
+	RS::EnvironmentAmbientSource ambient_source = RS::ENV_AMBIENT_SOURCE_BG;
+
+public:
+	// BG
 	RID sky;
 	float sky_custom_fov = 0.0;
 	Basis sky_orientation;
 	Color bg_color;
 	float bg_energy = 1.0;
 	int canvas_max_layer = 0;
-	RS::EnvironmentAmbientSource ambient_source = RS::ENV_AMBIENT_SOURCE_BG;
 	Color ambient_light;
 	float ambient_light_energy = 1.0;
 	float ambient_sky_contribution = 1.0;
 	RS::EnvironmentReflectionSource reflection_source = RS::ENV_REFLECTION_SOURCE_BG;
 	Color ao_color;
 
+	Basis get_sky_orientation() const { return sky_orientation; };
```

Comment:
I dont understand the benefit of exposing getters and setters like this for environment when the properties are also publicly accessible. Right now it looks like a change with no real benefit

Reply:
Well the idea is to make them private but I'm really just following convention here. I'm all for just accessing the members directly and removing the getters/setters.

Reply:
Changed this, indeed no need to over complicate this.